src/utils/strings.js

```diff
@@ -10,4 +10,41 @@ function snakeCaseToStartCase(str) {
     .join(' ');
 }
 
-export { snakeCaseToStartCase };
+/**
+ * Formats multiple session numbers into a string
```

Comment:
Maybe add a comment that it sorts the list (of integers only) and what it does to consecutive ranges.

Serves you right for creating a jsdoc.

Reply:
NEVER AGAIN!

Reply:
done